.teamcity/src/main/kotlin/common/performance-test-extensions.kt

```diff
@@ -42,6 +42,8 @@ fun performanceTestCommandLine(task: String, baselines: String, extraParameters:
     "-PperformanceBaselines=$baselines",
     "-PtestJavaVersion=${os.perfTestJavaVersion.major}",
     "-PtestJavaVendor=${os.perfTestJavaVendor}",
+    "-PautoDownloadAndroidStudio=true",
```

Comment:
Ideally we would setup this just for RealLifeAndroidStudioPerformanceTest

Reply:
Checking this value tasks value should do the trick

Reply:
I think it is fine to always pass this flag. What I think should happen is that the test tasks for the Android template projects should depend on the android studio download.